tools/policy_table_validator/main.cpp

```diff
@@ -43,6 +50,17 @@ int main(int argc, char** argv) {
     help();
     exit(MISSED_FILE_NAME);
   }
+
+#ifdef ENABLE_LOG
+  // --------------------------------------------------------------------------
+  // Logger initialization
+  // Redefince for each paticular logger implementation
+  auto logger = std::unique_ptr<logger::Log4CXXLogger>(new logger::Log4CXXLogger("log4cxx.properties"));
+  auto logger_impl = std::unique_ptr<logger::LoggerImp>(new logger::LoggerImp());
+  logger::Logger::instance(logger_impl.get());
+  logger_impl->Init(std::move(logger));
+#endif  // ENABLE_LOG
```

Comment:
@VladSemenyuk can be moved in a separate macro to logger abstraction as I mentioned above